### PR TITLE
Remove NOP1..10

### PR DESCRIPTION
**Pull request by constantineatdaglabs**
Fixes https://github.com/kaspanet/kaspad/issues/1616

### CODE REVIEW

File: domain/consensus/utils/txscript/data/script_tests.json [hunk 1]
```diff
@@ -4847,12 +4733,6 @@
     "SIG_PUSHONLY",
     "Tests for Script.IsPushOnly()"
   ],
-  [
-    "NOP1 0x01 1",
-    "BLAKE2B 0x20 0xda1745e9b549bd0bfa1a569971c77eba30cd5a4b EQUAL",
```
Comment (someone235) [inline domain/consensus/utils/txscript/data/script_tests.json:4852]: Why did you remove it?

**Review by someone235 — CHANGES_REQUESTED** [review]

File: domain/consensus/utils/txscript/data/script_tests.json (hunk 1 above)
Comment (constantineatdaglabs) [reply domain/consensus/utils/txscript/data/script_tests.json:4852]: Yes, because we have this just before this
```
  [
    "NOP 0x01 1",
    "BLAKE2B 0x20 0xda1745e9b549bd0bfa1a569971c77eba30cd5a4b EQUAL",
    "",
    "SIG_PUSHONLY",
    "Tests for Script.IsPushOnly()"
  ],
```

**Review by someone235 — APPROVED** [review]
